config/jobs/kubernetes/sig-instrumentation/sig-instrumentation-presubmit.yaml

```diff
@@ -0,0 +1,137 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-kind-json-logging
+    skip_branches:
+    - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-dashboards: sig-instrumentation-presubmits
+      testgrid-tab-name: pr-json-logging
+    decorate: true
+    # Not relevant for most PRs.
+    always_run: false
+    # This covers most of the code related to structured, contextual logging.
+    run_if_changed: /staging/src/k8s.io/component-base/logs/
+    # The tests might still be flaky (alpha features!) or this job might get triggered accidentally for
+    # an unrelated PR.
+    optional: true
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/krte:v20230406-23cb1879e3-master
+        command:
+        - wrapper.sh
+        - bash
+        - -c
+        # The modified e2e-k8s.sh with support for FEATURE_GATES and RUNTIME_CONFIG comes from
+        # https://github.com/kubernetes-sigs/kind/pull/3023. Once that PR is merged and a
```

Comment:
Let's focus on merging the PR first. Such workarounds have tendency to be forgotten and stick much longer than originally expected as merging the proper fix is no longer on critical path.

Reply:
I have https://github.com/kubernetes-sigs/kind/pull/3023 in my PR dashboard, so I won't forget - I just need to get busy reviewers to look at it again :cry:

Reply:
https://github.com/kubernetes-sigs/kind/pull/3023 was merged. Now we need to wait until it is included in https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz - probably in v0.19.0 (https://github.com/kubernetes-sigs/kind/tags).

Reply:
I think we can bind to commit as long it's within the original repo.

Reply:
Or we merge with the updated curl invocation which pulls https://github.com/kubernetes-sigs/kind/raw/c8618199912e3949aee0d7a8b966b462e4fb6f0e/hack/ci/e2e-k8s.sh